lib/linguist/heuristics.yml

```diff
@@ -139,6 +139,13 @@ disambiguations:
     and:
     - pattern: '(?i)^[ \t]*return '
     - negative_pattern: '(?i)[ \t]*gosub '
+- extensions: ['.bru']
```

Comment:
This is unnecessary and won't have any effect as this is the only language with this extension. Please remove this and the test.